src/components/AdminSettings/GeneralSettings.vue

```diff
@@ -24,6 +24,17 @@
 	<div id="general_settings" class="videocalls section">
 		<h2>{{ t('spreed', 'General settings') }}</h2>
 
+		<p>
+			<input id="ask_guest_username"
+				v-model="askGuestUsername"
+				type="checkbox"
+				name="ask_guest_username"
+				class="checkbox"
+				:disabled="loading || loadingAskGuestUsername"
+				@change="saveAskGuestUsername">
+			<label for="ask_guest_username">{{ t('spreed', 'Ask guest for a displayname') }} <span class="icon-info" :title="t('spreed', 'If guest have no display name, nextcloud will ask it on guest joining to conversation')" /></label>
```

Comment:
```suggestion
			<label for="ask_guest_username">{{ t('spreed', 'Ask guest for a displayname') }}</label>
			<em>{{ t('spreed', 'If guest have no display name, nextcloud will ask it on guest joining to conversation') }}</em>
```

Reply:
Is that how it should be?
![image](https://user-images.githubusercontent.com/18211871/115294554-68f94b80-a161-11eb-8919-a04c3fc0c80c.png)

Reply:
I added paragraph/ Now it looks better.